src/types/state.rs

```diff
@@ -13,10 +13,6 @@ pub enum State {
     Discharging,
     Empty,
     Full,
-
-    // Awaiting for https://github.com/rust-lang/rust/issues/44109
-    #[doc(hidden)]
-    __Nonexhaustive,
```

Comment:
Did you intentionally not add `#[non_exhaustive]` here?
I'm fine either way – just wanted to make sure it was on purpose.

Reply:
It's intentional. This matches the changes discussed in #2.

It looks like the library consumers that recently hopped-on would also want to handle additional enum values explicitly as they are added.